src/frontend/packages/suse-theme/loader/loading.css

```diff
@@ -1,6 +1,6 @@
 .stratos-initial-load-spinner-container {
   align-items: center;
-  background-color: #0D2C40;
+  background-color: #f2f2f2;
```

Comment:
The background for the loading page I think should be the same colour as the background of the log in/loading page
- transition wise it goes pre-loading page light grey --> log in/loading page dark green
- if user is logged in it then does a third colour hope light grey --> dark green --> light grey (most logged in pages)
- the pre-loading page is only around for a short time so can flicker, particularly jarring in the pre-login --> login --> logged in scenario